stage0/stop.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2016 The rkt Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//+build linux
+
+package stage0
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/coreos/rkt/common"
+
+	"github.com/appc/spec/schema/types"
+)
+
+func StopPod(dir string, force bool, uuid *types.UUID) error {
+	s1v, err := getStage1InterfaceVersion(dir)
+	if err != nil {
+		return fmt.Errorf("error determining stage1 interface version: %v", err)
+	}
+
+	if !interfaceVersionSupportsStop(s1v) {
+		return fmt.Errorf("stop entrypoint not supported by stage1")
+	}
```

Comment:
I think lines 31 to 39 could be deleted as discussed in https://github.com/coreos/rkt/pull/2438#discussion-diff-65917341